### ISSUE TITLE
Problems compiling with -DDOCS=ON

### ISSUE THREAD

**Issue by jamesb93** [flucoma/flucoma-cli#19]
Configuring CMake works in the build directory by running something like:

`cmake -GNinja -DFLUID_PATH=../../core -DDOCS=ON ..`

However, calling `ninja` afterwards produces this error:

```
[0/2] Re-checking globbed directories...
ninja: error: 'MAKE_CLI_REF', needed by 'CMakeFiles/CLI_MAKE_DOCS', missing and no known rule to make it
```

I've torn my hair out a bit searching for how the other environments deal with it and I can't see a difference... Max's counterpart is even directly commented out https://github.com/flucoma/flucoma-max/blob/5dbca51bbab24cd0861456cea5043c887457c3f6/CMakeLists.txt#L96. I would try to solve this one but any chance you take a look for me @weefuzzy . My CMake-fu is pushed beyond its limits 😿

**Comment by weefuzzy** [comment]
It's probably been excised from the current flucoma-docs. Will prod.

**Comment by weefuzzy** [comment]
Yeah, it's been commented out on the last line of CMakeLists for flucoma-docs
